websites/O/ogladajanime/presence.ts

```diff
@@ -0,0 +1,43 @@
+const presence = new Presence({ clientId: "1137362720254074972" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	anime: HTMLElement = document.querySelector("#anime_name_id"),
+	animeicon = document.querySelector(".img-fluid.lozad");
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		startTimestamp: browsingTimestamp,
+		largeImageKey: "https://i.imgur.com/xHce23t.png",
+	};
+
+	if (document.location.pathname === "/main2")
+		presenceData.details = "Viewing the home page";
+	else if (document.location.pathname.includes("/search/name/"))
+		presenceData.details = "Searching anime";
```

Comment:
Considering the language of the website, strings and metadata should be written [with support for the main language](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets) of this website.

Reply:
should I understand that strings and metadata are to be written in the language of the site?